src/components/Navbar/navbar.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import style from "./navbar.module.css";
+
+export default function NavBar() {
+  return (
+    <header className={style.navigation}>
+    <ul> 
+        <li><a href="#home">Home</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#portfolio">Portfolio</a></li>
+        <li><a href="#contact">Contact</a></li>
```

Comment:
Nice job with navigation. Make sure to use the `Link` tag stead of the `a` for routing.
e.g.` <Link href="/resume"></Link>`.  Be sure to `import Link from "next/link"`